se/epub.py

```diff
@@ -34,7 +34,8 @@ def convert_toc_to_ncx(epub_root_absolute_path: Path, toc_filename: str, xsl_fil
 
 	toc_tree = se.easy_xml.EasyXhtmlTree(xhtml)
 	transform = etree.XSLT(etree.parse(str(xsl_filename)))
-	ncx_tree = transform(etree.fromstring(str.encode(xhtml)), cwd=f"'{epub_root_absolute_path}{os.path.sep}'")
+	posix_path = Path(epub_root_absolute_path ).as_posix() 
```

Comment:
`epub_root_absolute_path` is already a PosixPath, see the function signature above, so it's not necessary to convert it. So, you can just inline this directly into the f-string without needing a new object or additional variable.

Reply:
Pushed another commit that inlines it. Sorry about that.